caffe_app/views/import_prototxt.py

```diff
@@ -2,25 +2,31 @@
 from django.views.decorators.csrf import csrf_exempt
 from django.http import JsonResponse
 import yaml
+import os
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
 from caffe.proto import caffe_pb2
 from google.protobuf import text_format
 
 @csrf_exempt
 def importPrototxt(request):
     if request.method == 'POST':
-        try:
-            prototxt = request.FILES['file']
-        except Exception:
-            return JsonResponse({'result': 'error', 'error': 'No Prototxt model file found'})
-
+        if 'file' in request.FILES:
+            try:
+                prototxt = request.FILES['file']
+            except Exception:
+                return JsonResponse({'result': 'error', 'error': 'No Prototxt model file found'})
```

Comment:
Can you please change the response code to `{'error': 'No Prototxt model file found'}` in case of any error. The current key value pair `'result': 'error'` is just duplication.